Rubrik/Public/Protect-RubrikTag.ps1

```diff
@@ -84,12 +84,7 @@ function Protect-RubrikTag
   Process {
 
     #region One-off
-    Write-Verbose -Message 'Gathering the SLA Domain id'
-    if (!$SLA -and !$Inherit -and !$DoNotProtect) 
-    {
-      $SLA = $Tag
-    }
-    $SLAID = Test-RubrikSLA -SLA $SLA -Inherit $Inherit -DoNotProtect $DoNotProtect    
+    $SLAID = Test-RubrikSLA -SLA $SLA -Inherit $Inherit -DoNotProtect $DoNotProtect
```

Comment:
Unclear on this, it was my understanding that -Inherit and -DoNotProtect were switch parameters. They don't appear defined in Test-RubrikSLA. Should we define their use and what type of values they use to match up with the use case here?